plugin/pkg/auth/authenticator/request/basicauth/basicauth.go

```diff
@@ -17,10 +17,13 @@ limitations under the License.
 package basicauth
 
 import (
-	"net/http"
-
+	"errors"
+	"github.com/golang/glog"
+	"github.com/msteinert/pam"
```

Comment:
All packages have to be vendored in godeps. Add this package in a separate commit